wurst/systems/units/hostiles/TheOne.wurst

```diff
@@ -0,0 +1,34 @@
+package TheOne
+import BonyAnimal
+
+
+public class TheOne extends BonyAnimal
+    private static let unitTypeId = UNIT_MAMMOTH
+
+    static function initialize()
+        register(unitTypeId, mammoth -> new TheOne(mammoth))
+
+    construct(unit whichUnit)
+        super(whichUnit)
+
+    construct(player whichPlayer, vec2 pos)
+        super(whichPlayer, pos)
+
+    construct(player whichPlayer, vec2 pos, angle facing)
+        super(whichPlayer, pos, facing)
+
+    override function postCreate()
+        super.postCreate()
+        drops.add(ITEM_ESSENCE_BEES)
+
```

Comment:
While a nerf is in place, just dropping an eob and nothing else seems a bit light since he doesnt seem to give a ton of exp either